### PR TITLE
LPS-124252 [Follow-up] Create traffic source detail representation for referral traffic

### PR DESCRIPTION
**Pull request by darquesdev**
**Motivation**:
This is an improvement for https://github.com/liferay-tango/liferay-portal/pull/624. The motivation is to make the method `_getTrafficSources` of `AnalyticsReportsProvider` public. This way all the AC calls are exposed.

**Solution**:
[Change the method visibility
](https://github.com/liferay-tango/liferay-portal/pull/635/commits/f740b7b5c2ccbc354a661f64ff723aca9178a44d)

**How to test it**:
[Unit tests have been added](https://github.com/liferay-tango/liferay-portal/pull/635/commits/1ab2990ef3bb26bf75c4d445edf3b14f7d822a36)

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

File: modules/dxp/apps/analytics/analytics-reports-web/src/test/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProviderTest.java [hunk 1]
```diff
@@ -422,6 +423,168 @@ public void testGetTrafficChannelsWithCountrySearchKeywords()
...
+			String.valueOf(trafficSources.get("organic")));
+		Assert.assertEquals(
+			String.valueOf(
+				new TrafficSource(
+					Collections.emptyList(), "paid", 2010L, 44.1D)),
+			String.valueOf(trafficSources.get("paid")));
+	}
+
+	@Test
+	public void testGetTrafficSourcesWithAsahFaroBackendError()
```
Comment (darquesdev) [inline modules/dxp/apps/analytics/analytics-reports-web/src/test/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProviderTest.java:469]: Hey @cgoncas, I decided to leave the exception handling as it was to not change the current behavior, let me know what you think. Thx!

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

**Review by cgoncas — APPROVED** [review]
LGTM

**Comment by darquesdev** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**